mlir/utils/jenkins/Jenkinsfile

```diff
@@ -793,5 +817,79 @@ pipeline {
                 }
             }
         }
+        stage ("MIGraphX") {
+            when {
+                beforeAgent true;
+                equals expected: true, actual: params.checkMIGraphX;
+                equals expected: true, actual: params.nightly;
+            }
+            matrix {
+                axes {
+                    axis {
+                        name 'CODEPATH'
+                        values 'mfma', 'vanilla', 'navi21'
+                    }
+                }
+                agent {
+                    docker {
+                        image dockerImage()
+                        args dockerArgs()
+                        label getLabelFromCodepath("${CODEPATH}")
+                        alwaysPull true
+                    }
+                }
+                when {
+                    beforeAgent true
+                    expression { return shouldRunFromCodepath("$CODEPATH") }
+                }
+                environment {
+                    HOME="${WORKSPACE}"
+                }
+                stages {
+                    stage('Environment') {
+                        steps {
+                            echo "codepath is ${CODEPATH}"
+                            showEnv()
+                        }
+                    }
+                    stage("Install MIGraphX Dependencies") {
+                        steps {
+                            // We are installing miopen-hip here (and not in Dockerfile) to avoid collision with MIOpen checks above
+                            sh 'apt-get install miopen-hip'
```

Comment:
As an alternative, maybe use or create a build of MIOpen here and add it to $PATH? (Ex. If this were moved onto the same node as the perf test to save on MIOpen rebuilds)

Reply:
_Or_ we could create a second Docker that inherits from the first but includes MIOpen and the MIGraphX dependencies

Reply:
> As an alternative, maybe use or create a build of MIOpen here and add it to $PATH? (Ex. If this were moved onto the same node as the perf test to save on MIOpen rebuilds)

apt-get install does not do a build anyway, right ?

Reply:
Wait, the MIOpen with HIP does a make install? Who added that??

And another advantage to adding a second Docker image is that we don't have to `wget` Resnet50 every time